en/advanced_config/esc_calibration.md

```diff
@@ -2,20 +2,34 @@
 
 :::note
 These instructions are only relevant to [PWM ESCs](../peripherals/pwm_escs_and_servo.md) and [OneShot ESCs](../peripherals/oneshot.md).
-[DShot](../peripherals/dshot.md) and [CAN](../can/README.md) ESCs ([DroneCAN](../dronecan/escs.md)/Cyphal) do not require calibration.
+[DShot](../peripherals/dshot.md) and [CAN](../can/README.md) ESCs ([DroneCAN](../dronecan/escs.md)/Cyphal) do not require this kind of calibration.
 :::
 
 Electronic Speed Controllers (ESCs) regulate motor speed (and direction) based on the PWM input value from the flight controller (FC).
 The range of inputs to which an ESC will respond is configurable, and the default range can differ even between ESCs of the same model.
 
-This calibration updates all the ESCs with the maximum and minimum PWM input values that will be supplied by the flight controller.
-Subsequently all the ESCs/motors will respond to flight controller input in the same way (across the whole input range).
+This calibration updates all the ESCs with a fixed maximum (2000us) and minimum (1000us) PWM input from the flight controller.
+Subsequently all the ESCs/motors will respond to flight controller input in the same way across the whole input range.
 
-Calibration is recommended for all ESCs, and in particular for low cost models.
+Calibration is recommended for all PWM/OneShot ESCs that support it, and in particular for low cost models.
+
+:::note
+High-quality controllers come with a factory calibration and should all respond the same way.
+In practice the input range may differ even on high quality controllers because they may be have been manually recalibrated after leaving the factory, and that might not have been done correctly. Following manual calibration they might not respond in the same way as they did after factory calibration.
+
+Using this calibration tool is recommended, as it ensures that all ESCs behave exactly the same way.
+:::
+
+:::note
+Some ESCs do not support such a calibration. They must come factory calibrated and should respond consistently out of the box.
+This has to be verified using [Actuator Testing](../config/actuators.md#actuator-testing). Jump to step 7 which is still important.
+:::
+
+OneShot ESCs should be [configured to use OneShot](../peripherals/oneshot.md#px4-configuration) before calibration otherwise calibration steps should be repeated after switching.
 
 ## Preconditions
 
-The system must include a power module (PX4 uses the measured voltage to determine whether or not a battery is connected).
+You must be able to (un)power ESCs and flight controller independently. If the flight controller is powered through USB and there is a power module the procedure can detect battery connection. If battery power can't be detected e.g. because there's no power module then the calibration sequence is still performed based on timeouts.

```

Comment:
```suggestion
You must be able to (un)power ESCs and flight controller independently.
If the flight controller is powered through USB and there is a power module the procedure can detect battery connection.
If battery power can't be detected e.g. because there's no power module then the calibration sequence is still performed based on timeouts.

```